notebook/static/notebook/js/keyboardmanager.js

```diff
@@ -10,7 +10,7 @@
 
 define([
     'base/js/utils',
-    'base/js/keyboard',
+    'base/js/keyboard'
```

Comment:
Sorry, tried enabling `eslint` and my editor complains. I can deactivate and revert that.